### ISSUE TITLE
2.1.14 - Wireshark doesn't capture any more

### ISSUE THREAD

**Issue by byunkook** [GNS3/gns3-gui#2746]
OS: Windows 10 10.0.17763.379
GNS3: 2.1.4
WIreshark: 2.6.5

I was trying to capture traffic between vmware kali linux vm and a router and I noticed "no packets" in wireshark.

I had tiny core appliance also connected to the router and the capture worked fine between router and the appliance.

I just created a brand new project and tried to capture between vpcs get no packets message as shown below.

![2019-03-23_20-44-31](https://user-images.githubusercontent.com/46533861/54873522-24e2c380-4dae-11e9-93b2-f434077de641.png)

**Comment by byunkook** [comment]
After further test with different scenarios, I found where the issue is.

Wireshark cannot capture if one of devices is from local server.

VPC1(local) - VPC2(local) - no packets
VPC1(local) - VPC3(GNS3) - no packets
VPC3(GNS3) - VPC4(GNS3) - wireshark can capture.

**Comment by cioby23** [comment]
Please have a look at this issue https://github.com/GNS3/gns3-gui/issues/2722. You might experience a similar problem. Try and remove NpCAP is you have installed.

**Comment by byunkook** [comment]
I upgraded GNS3 to the 2.1.15 and looks like that fixed the issue.

This is the complete steps I performed.

Uninstalled wireshark I upgraded to 3.0
Uninstalled Virtualbox.
Upgraded GNS3 to 2.1.15. wireshark was installed from GNS3 installer.
Reinstalled Virtualbox
I still have npcap 0.99-r9 installed in my system. I never uninstalled/reinstalled this.

Now I can capture betwen localserver VPCs, between VMware and gns3VM server device, and between virtualbox and gns3VM server device
